server/src/Settings.php

```diff
@@ -89,7 +89,7 @@ public static function getDBPassword() : string{
 	}
 
 	public static function getDBSSL() : bool{
-		return in_array(getenv('MYSQL_SSL', true), ['true', '1'], true) && @filesize(self::getDBSSLCA());
```

Comment:
**Notes**:
- If SSL files are not found then `MYSQL_SSL` should be disabled by default.
- `MYSQL_SSL` when enabled its value in docker is `"true"` but outside is `"1"`.

Reply:
Even if I enable SSL with default path to certificates, it doesn't get enabled. So this doesn't work as it should.

Reply:
I think that I found an issue with this outside of the docker, so I made the proper changes. ^_^

Reply:
I also don't think this should fall to false if file is incorrect or it doesn't exists as then user will think that they are using secure connection, but in reality it has been silently disabled in the back end.

Reply:
In this case it would be better for the Error to be thrown or connection to a database unsuccessful.

Reply:
OK